applications/OptimizationApplication/python_scripts/utilities/optimization_info.py

```diff
@@ -1,145 +1,594 @@
-import KratosMultiphysics as Kratos
+from __future__ import annotations
```

Comment:
i can make my self live with OptimizationInfo and OptimizationData but hoenestly it is very confusing, even the name of the file is optimization_info, why do not you merge them into single class ?! what are the reasons ?

Reply:
It cannot be merged into one class. `OptimizationData` is a recursive data holder which can hold itself as well. So it always needs to be a separate class. `OptimizationInfo` is holding an `OptimizationData` which is passed around every where.

Reply:
Moved to a different file to avoid confusion.